cmd/deploy.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/docker/libcompose/docker"
+	"github.com/docker/libcompose/docker/ctx"
+	"github.com/docker/libcompose/project"
+	"github.com/spf13/cobra"
+)
+
+// deployCmd represents the up command
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "Trigger an image deployment for all shipments and containers defined in compose files",
+	Long:  "Note that the deploy command is a subset of the up command without updates for environment variables, replicas, barge info, etc.",
+	Run:   deploy,
+}
+
+var shipmentBuildToken string
+
+func init() {
+	deployCmd.PersistentFlags().StringVarP(&shipmentBuildToken, "token", "t", "", "the shipment build token to use")
```

Comment:
So, @jkurz, based on our prior discussion, it sounds like we could greatly simply things by implementing a required naming convention for shipment/env build token environment variables.  This way, a single `docker-compose.yml` and `harbor-compose.yml` could be used to deploy a multi-shipment/multi-container app to an environment with a single command.

For example, with a `harbor-compose.yml` that looks something like...

```
version: "1"
shipments:
  shipment1:
    env: dev
    containers:
      - container1
      - container2
  shipment2:
    env: dev
    containers:
      - container3
      - container4 
```

and the following env vars defined in your build... 

```
SHIPMENT1_DEV_TOKEN
SHIPMENT2_DEV_TOKEN
```

you could deploy the shipments using:

```
harbor-compose deploy
```

and if you wanted to deploy the shipments to QA, you could add the following env vars and then do:

```
SHIPMENT1_QA_TOKEN
SHIPMENT2_QA_TOKEN
```
```
harbor-compose deploy --env qa
```

...all with a single harbor-compose.yml.  And your CI/CD build scripts would be nice and clean...

- set image versions (either as env vars or defined in docker-compose.yml)
- docker-compose build
- docker-compose up -d && ./test
- docker-compose push
- harbor-compose deploy

...and all of your container images would be built, tested, pushed to a registry, cataloged and deployed.

I think that could work.